binsync/decompilers/ida/compat.py

```diff
@@ -795,6 +795,8 @@ def has_older_hexrays_version():
 
     return not vers.startswith("8.2")
 
-
-
-
+def save_idb():
+    binary_name = get_binary_path().split("/")[-1]
+    target_name = binary_name + ".i64"
```

Comment:
little sketchy, but I think it's fine. Just make sure to check the bit version so you save with the right name.